### ISSUE TITLE
nginxServerConfig dollar signs removed

### ISSUE THREAD

**Issue by naterich2** [zodern/meteor-up#965]
Mup version (`mup --version`): 1.4.5

I am trying to proxy to s3 for one of the paths on my server:

```
location /reports {
    resolver 172.16.0.23 valid=300s;
    resolver_timeout 10s
    set $s3_bucket 'myBucket.s3.amazonaws.com'
    rewrite ^/reports/(.*)$ /$1 permanent;

    proxy_intercept_errors 	on;
    proxy_redirect 		off;
    proxy_set_header 	Host $s3_bucket;
    proxy_set_header 	X-Real-IP $remote_addr;
    proxy_set_header 	X-Forwarded-For $proxy_add_x_forwarded_for; 
    proxy_hide_header 	x-amz-id-2;
    proxy_hide_header	x-amz-request-id;
}
```
The problem is, the config that actually goes on the docker when running ```mup setup``` gets changed:
```
docker exec -t -i mup-nginx-proxy /bin/bash
cat /etc/nginx/vhost.d/mySite.com

location /reports {
    resolver 172.16.0.23 valid=300s;
    resolver_timeout 10s
    set  'myBucket.s3.amazonaws.com'
    rewrite ^/reports/(.*)$ / permanent;

    proxy_intercept_errors 	on;
    proxy_redirect 		off;
    proxy_set_header 	Host ;
    proxy_set_header 	X-Real-IP ;
    proxy_set_header 	X-Forwarded-For ; 
    proxy_hide_header 	x-amz-id-2;
    proxy_hide_header	x-amz-request-id;

}

```
All the dollar signs are removed except for in the regexp.  I wasn't sure if this was because of a string literal thing, so I tried escaping with ```$$```, but that didn't work.  I also looked through the code and saw that ejs is being used to run the scripts, I search issues on that repository and saw a suggestion to try escaping with ``` <%- "$s3_bucket" %> ``` but that didn't work either.  I'm not very familiar with esj, but I think something in that is causing the symbol and word to be removed.  
    
 
This is causing my nginx to not work.  Any ideas how to fix this?

**Comment by jramer** [comment]
`nginx-config.sh` removes them as it's a shell script, It works if you escape them with `\`.
e.g.
```
rewrite ^/reports/(.*)\$ /\$1 permanent;
```

**Comment by zodern** [comment]
This is fixed in the latest [1.5 betas](https://github.com/zodern/meteor-up/pull/976)
